src/digitaltwin/get_data_from_apis.py

```diff
@@ -42,25 +44,20 @@ def check_table_exist(engine, source_list):
     return tables_in_db, tables_not_in_db
 
 
-def wfs_request_from_db(engine, tables, polygon):
+def wfs_request_from_db(engine, table_names, polygon):
     """Make wfs request for the requested sources.
-    tables is source_name ('_50329-nz-road-centrelines',
+    table_names is source_name ('_50329-nz-road-centrelines',
                            '_50319-nz-railway-centrelines')
     """
-    db_tbl = wfs_request.access_api_info(engine, tuple(tables))
+    db_tbl = wfs_request.access_api_info(engine, tuple(table_names))
     for i in range(len(db_tbl)):
         data_provider = db_tbl.loc[i, "data_provider"]
         layer = db_tbl.loc[i, "layer"]
         base_url = db_tbl.loc[i, "source_api"]
         geometry_name = db_tbl.loc[i, "geometry_col_name"]
         table_name = db_tbl.loc[i, "source_name"]
-        keys = engine.execute(
-            "select key from api_keys where data_provider = %(data_provider)s",
-            ({"data_provider": data_provider}),
-        )
-        key = ""
-        for k in keys:
-            key = key + k[0]
+        key_name = f"{data_provider}_API_KEY"
```

Comment:
Great! No longer need to manually insert an api_keys table into the database, and make great use of the .env file :)